src/cli.py

```diff
@@ -20,6 +20,7 @@
 '''  to run > sudo LD_LIBRARY_PATH=../libcsp/build PYTHONPATH=../libcsp/build python3 src/cli.py -I uart -d /dev/ttyUSB1  '''
 import time
 from groundStation import groundStation
+import traceback
```

Comment:
unused include

Reply:
Used for testing purposes. Added comment